includes/wsuwp-shortcode-fundselector.js

```diff
@@ -138,17 +138,19 @@ jQuery(document).ready(function($) {
 
 		if($parent.hasClass("fund-scholarship")) $("#genScholarship").prop("checked", false);
 
-		$parent.remove();
+		$parent.fadeOut(1000, function(){ 
+            $(this).remove(); 
```

Comment:
I think only the remove() function needs to be inside the fadeOut. The rest of it can occur at the same time.
In particular, we should disable and hide the Continue button right away, so they can't click it during the 1 sec transition with no funds.